main/getting-started/alpha.md

```diff
@@ -0,0 +1,60 @@
+# Agoric Alpha	
+Release Version: Adriatic
+
+![Alpha Cover](./assets/alpha-cover.png)
+
+We’re building components to power the DeFi ecosystem. Come take a peek under the hood and help us scale our next-gen infrastructure. The Alpha Adriatic release provides you access to our toolset, API, and public chain.
+
+[Start Coding](https://agoric.com/documentation/getting-started/)
+[Join Community](https://discord.gg/gC9z6US)
+
+---------------------
+
+## What you can build in Alpha Adriatic 
+
+Numerous transaction components -- what we think of as DeFi Legos -- are available in the Alpha Adriatic release. We’ll be adding to the pool with our growing community’s help and participation in the coming months as we scale contributions. The initial slate of elements includes the following: 
+
+![Automated Market Maker](./assets/amm.png)
+### Automated Market Maker
+Build a smart contract that acts as an on-chain automated DEX with liquidity. 
+
+![Autoswap](./assets/autoswap.png)
+### Autoswap
+Tap into our Uniswap-like, constant-product AMM that executes upon finding a match in the current liquidity pool.
```

Comment:
Unless you're sure all of the audience will know what it is/stands for (I don't), give an expansion for AMM i.e. "AMM (Automatic Metal Model)"